projects/plugins/crm/modules/mailpoet/includes/class-mailpoet-background-sync-job.php

```diff
@@ -334,18 +329,17 @@ public function import_local_subscribers( $page_no = -1 ) {
 
 		}
 
-
 		// check for completion
 		$total_page_count = $this->get_total_page_count();
-		if ( $page_no >= $total_page_count ) {
+		if ( $page_no >= ( $total_page_count - 1 ) ) {
```

Comment:
`$page_no` works in the `0..(total_pages - 1)` range and `$total_page_count` in the `1..total_pages` range.